### PR TITLE
terraform: configure terraform cloud

### PR DESCRIPTION
**Pull request by zimbatm**
> Please don't merge this yet. This needs to be coordinated to import all the terraform state of the existing resources before. 

Continuing on the road of consolidation and transparency. This maps the current resources that we have.

**Status:** open for review

### CODE REVIEW

File: terraform/terraform_cloud.tf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#
+# Terraform Cloud is used only for one thing: to store the terraform state.
+#
+locals {
+  # FIXME: add all the admins of the org
+  tfe_owners = {
+    zimbatm = "zimbatm@zimbatm.com"
+  }
+
+  tfe_org = tfe_organization.nix-communiyt.name
```
Comment (zowoq) [inline terraform/terraform_cloud.tf:11]:
```suggestion
  tfe_org = tfe_organization.nix-community.name
```

File: terraform/terraform_cloud.tf [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+# Org setup
+resource "tfe_organization" "nix-community" {
+  name = "nix-community"
+  # FIXME: host our own email
```
Comment (zowoq) [inline terraform/terraform_cloud.tf:17]: Do you mean running an email server ourselves?

**Comment by zowoq** [comment]
Thanks for this, tf cloud access was something I was planning to ask about, this looks like a good way to manage it.

File: terraform/terraform_cloud.tf (hunk 2 above)
Comment (zimbatm) [reply terraform/terraform_cloud.tf:17]: Hopefully not. A forwarding address service is enough.

**Comment by zimbatm** [comment]
I'm running out of time to fix this so I will merge the half solution and get back to it tomorrow.

I will need all of your email addresses to add to the list of members.

bors merge

**Comment by zimbatm** [comment]
bors merge

**Comment by zimbatm** [comment]
bors merge

**Pull request merged**